src/routes/Player/Player.js

```diff
@@ -554,6 +563,15 @@ const Player = ({ urlParams, queryParams }) => {
                     :
                     null
             }
+            {
+                optionsMenuOpen ?
+                    <OptionsMenu
+                        className={classnames(styles['layer'], styles['menu-layer'])}
+                        stream={videoState.stream}
```

Comment:
player.selected.stream might be better here, because when the video throws error, videoState.stream will be null

Reply:
Oh okay !